webcompat/form.py

```diff
@@ -100,6 +114,15 @@ def get_problem(category):
     return u'Unknown'
 
 
+def get_browser_test(category):
```

Comment:
ok, I agree it should probably be one function. but will wait for further decision.